CARTOGRAPHY.md

```diff
@@ -37,3 +37,53 @@ account.
+
+## Design guidelines
+
+The following guidelines have been found to be useful design principles that can help creating a better map in line with the above goals.  Keep in mind that none of these principles represents a goal on its own and most of them are highly condensed summaries of more complex considerations.
+
+* The difference in rendering between different types of features should foremost be based on their difference in meaning and purpose for the target map users.  Differences in physical appearance are only relevant as far as they also represent differences in meaning and purpose.
+
+* Design of symbols, area and line patterns should be in a way that avoids their geometries to be mistaken as actual data.
+
+* The connection between data and appearance of features in the map needs to be comprehensible for the viewer.
+
+* When changes have a strong impact on the overall appearance of the map input should be sought from the broader OpenStreetMap community and particular consideration given to their comments and concerns.
+
+### Colors
+
+* Differentiating features through different colors only makes sense if this difference can actually be read by the map users.
+
+* For area colors we prefer light and low saturation tones.  Lines, individual symbols and labels also can be stronger in color.
+
+* Stronger colors indicate fairly distinct and meaningful features as opposed to more general and less distinct features which use weaker colors.
+
+* Small areas work well with somewhat stronger colors while large areas tend to call for weaker tones.
+
+* Symbols of area patterns need to be relatively weak in color with little contrast towards the base color so they cannot be mistaken for an individual symbol.
+
+* Area outlines should - when used - be in colors that are not too strong and harmonic with the fill color.  Care should be taken that area outlines are not mistaken for line features.
+
+* Use of blue for area and line colors should be reserved for water related features.  Symbols and labels in blue color can also be used for other purposes but sparsely and in a tone clearly distinct from all water related features
+
+* Use of green should be reserved for vegetation related features, in particular in case of area and line colors.
+
+* Use of red, orange, yellow and brown line colors should be reserved to roads and paths.
+
+* Transparency tends to lead to undesirable and ambiguous color mixing and should therefore be used very carefully.
+
+* It is generally advisable to design color relationships in perceptual color spaces.
+
+### Zoom levels
+
+* Styling of map elements should always work for all parts of earth shown on the map and for the whole range of map scales present in each zoom level the feature is shown in.
+
+* Design differences between zoom levels should be small.
```

Comment:
"Design differences between adjacent zoom levels should be small."

I am not sure about exact word (adjacent/joining/neighbouring are all really bad), but it should be clear that z5 and z19 may have vast differences in styling and it is OK. It is partially clarified in the next line but it would be better to make it completely clear.